QLogger.h

```diff
@@ -43,10 +43,13 @@ class QLoggerManager
     * @brief Gets an instance to the QLoggerManager.
     * @return A pointer to the instance.
     */
-   static QLoggerManager *getInstance();
+   static QLoggerManager &getInstance();
+
+   bool hasModules() const;
```

Comment:
Would be nice to have documentation as the other methods.

Reply:
Idem

Reply:
Same as above: please add doxygen.